src/app/server/Mdns.cpp

```diff
@@ -244,25 +232,50 @@ CHIP_ERROR Advertise(bool commissionableNode)
     return mdnsAdvertiser.Advertise(advertiseParameters);
 }
 
+/// Set MDNS commissioner advertisement
+CHIP_ERROR AdvertiseCommisioner()
+{
+    return Advertise(false, CommissioningMode::kDisabled);
+}
+
+/// Set MDNS commissionable node advertisement
+CHIP_ERROR AdvertiseCommissionableNode(CommissioningMode mode)
+{
+    return Advertise(true, mode);
+}
+
 /// (Re-)starts the minmdns server
+// Commissioning mode is enabled if device has not yet been commissioned
 void StartServer()
```

Comment:
If we're keeping this signature, why the changes to its various callers to call `StartServer(CommissioningMode::kDisabled)` directly?

Reply:
reverted Server.cpp usage to previous signature